GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+# Governance
+
+The Selenium Project wants as much as possible to operate using procedures that
+are fair, open, inviting, and ultimately good for the community. For that reason,
+we find it valuable to codify some of the ways that the Project goes about its
+day-to-day business. We want to make sure that no matter who you are, you have
+the opportunity to contribute to Selenium. We want to make sure that no
+corporation can exert undue influence on the community or hold the Project
+hostage. And likewise, we want to make sure that corporations which benefit
+from Selenium are also incentivized to give back. This document describes how
+various types of contributors work within the Selenium project.
+
+## Roles and Responsibilities
+
+### Users
+
+Users are community members who have a need for the project. Anyone can be a
+User; there are no special requirements. Common User contributions include:
+
+* Evangelizing the project (e.g., display a link on a website and raise
+awareness through word-of-mouth)
+* Informing strengths and weaknesses from a new user perspective 
+(e.g., raising a bug as a GitHub issue, proposing a new feature)
+* Providing moral support (a "thank you" goes a long way).
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such Users may find themselves becoming
+Contributors, as described in the next section.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project,
+most often in the form of code and/or documentation. Anyone can become a Contributor,
+and contributions can take many forms, e.g.:
+* Help other users through any of the [communication channels](#communication-channels)
+made for that purpose.
+* Triage GitHub issues.
+* Organize Selenium meetups.
+* Organize and collaborate in Selenium Conferences.
+
+There is no expectation of commitment to the project, no specific skill requirements,
+and no selection process.
+
+Some Contributors might have some basic privileges to the GitHub repos, based on
+their type of contribution (e.g., close an issue after triaging it).
+
+Contributors have read-only access to source code and submit changes via pull
+requests. Contributor pull requests have their contribution reviewed and merged
+by a TLC member or a Committer. TLC members and Committers work with Contributors
```

Comment:
referencing a non-standard acronym before it's defined. Change to -

    [Technical Leadership Committee (TLC)](#technical-leadership-committee-tlc)

Reply:
Good catch, thank you, fixed it.